server/src/main/java/org/opensearch/index/remote/RemoteStoreStatsTrackerFactory.java

```diff
@@ -68,7 +68,7 @@ public RemoteStoreStatsTrackerFactory(ClusterService clusterService, Settings se
 
     @Override
     public void afterIndexShardCreated(IndexShard indexShard) {
-        if (indexShard.indexSettings().isRemoteStoreEnabled() == false) {
+        if (indexShard.indexSettings().isRemoteStoreEnabled() == false && indexShard.indexSettings().isRemoteNode() == false) {
```

Comment:
consider this case - mixed mode cluster, non-remote index but remote attributes present on the node. This will lead to tracking remote stats for a non-remote index. Is this use case possible in mixed mode cluster?

Reply:
Yes, we would track stats for migrating index .